qiskit_experiments/calibration_management/save_utils.py

```diff
@@ -0,0 +1,253 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""A helper module to save calibration data to local storage.
+
+.. warning::
+
+    This module is expected to be internal and no stable user-facing API
+    may not be guaranteed.
+
+.. note::
+
+    These model version must be upgraded when :class:`.Calibrations` data structure changes
+    in future and when the current data model becames unable to support it.
+
+"""
+
+from dataclasses import dataclass, field, asdict
+from datetime import datetime
+from typing import List, Dict, Any
+
+from qiskit.pulse import ScheduleBlock
+
+from .calibrations import Calibrations
+from .control_channel_map import ControlChannelMap
+from .parameter_value import ParameterValue
+
+
+@dataclass(frozen=True)
+class ParameterModelV1:
+    """A data schema of a single calibrated parameter.
+
+    .. note::
+        This is intentionally agnostic to the data structure of
+        Qiskit Experiments Calibrations for portability.
+
+    """
+
+    param_name: str
+    qubits: List[int]
+    schedule: str = ""
+    value: float = 0.0
+    datetime: datetime = datetime.now()
```

Comment:
I wonder if `now()` as default is the right choice. It could be misleading depending but it depends on how this class is supposed to be used. Can you add a few sentences on expected usage in the docstring.

Reply:
That's fair point. I turned it into `None` as a default value in [993fcb8](https://github.com/Qiskit/qiskit-experiments/pull/1120/commits/993fcb878897fe892ffc33c6d389682fd1565e67). Missing default value will be an edge case if you create this entry from QE, because `ParameterValue` class provides a default value.